src/components/pages/devices/devicesGrid/devicesGrid.js

```diff
@@ -38,6 +38,19 @@ export class DevicesGrid extends Component {
     ];
   }
 
+  /**
+   * Get the grid api options
+   *
+   * @param {Object} gridReadyEvent An object containing access to the grid APIs
+  */
+  onGridReady = gridReadyEvent => {
+    this.deviceGridApi = gridReadyEvent.api;
+    // Call the onReady props if it exists
+    if (isFunc(this.props.onGridReady)) {
+      this.props.onGridReady(gridReadyEvent);
+    }
+  };
+
```

Comment:
Any reason for moving this method?

Reply:
I initially removed it but saw that it was still needed to added it back. I just put it back in a different location